pkg/agent/types/annotations.go

```diff
@@ -23,4 +23,7 @@ const (
 
 	// NodeWireGuardPublicAnnotationKey represents the key of the Node's WireGuard public key in the Annotations of the Node.
 	NodeWireGuardPublicAnnotationKey string = "node.antrea.io/wireguard-public-key"
+
+	// ServiceExternalIPPoolAnnotationKey represents the key of the Service's desired external IP pool.
+	ServiceExternalIPPoolAnnotationKey string = "service.antrea.io/external-ip-pool"
```

Comment:
I feel no need to add the "service" prefix. But @tnqn can comment what is our annotation key convention.